packages/autometrics-typescript-plugin/src/utils.ts

```diff
@@ -20,3 +20,16 @@ export function createLogger(project: ts.server.Project) {
     project.projectService.logger.info(`${PLUGIN_NAME}: ${msg}`);
   };
 }
+
+export function hasAutometricsDecorator(node: ts.Node) {
```

Comment:
maybe a nit but I'd be somewhat inclined to currently keep all functions that deal with AST processing in the `astHelpers.ts` file. This makes it somewhat easier to follow the AST parsing logic of the plugin

Reply:
Fair point, will move it to `astHelpers.ts` 👍